drivers/ethernet/eth_stm32_hal.c

```diff
@@ -32,19 +32,44 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #error DTCM for DMA buffer is activated but zephyr,dtcm is not present in dts
 #endif
 
+#ifndef PHY_ADDR
+#define PHY_ADDR	((uint32_t)CONFIG_ETH_STM32_HAL_PHY_ADDRESS)
+#endif
+
+#ifndef PHY_BSR
+#define PHY_BSR		((uint16_t)0x0001U)
```

Comment:
Constant should be documented/referenced

Reply:
The value of `PHY_BSR` comes from original implementation and without description/reference. Therefore I am not sure, I can describe it correctly.

Reply:
The corresponding define is copy-pasted from `stm32f7xx_hal_conf.h` as follows:
`#define PHY_LINKED_STATUS  ((uint16_t)0x0004U)  /*!< Valid link established */`

It defined only for `H7` series for sake of simplicity.